src/test/java/me/itzg/helpers/modrinth/ModrinthCommandTest.java

```diff
@@ -378,6 +380,38 @@ void handlesDatapacksLatestVersion(@TempDir Path tempDir) {
             .hasContent("content of zip");
     }
 
+    @Test
+    void usingListingFile(@TempDir Path tempDir) throws Exception {
+        setupStubs();
+
+        final Path listingFile = Files.write(tempDir.resolve("listing.txt"),
+            Arrays.asList(
+                "fabric-api",
+                "# This is a comment",
+                "cloth-config",
+                "",
+                "# Another comment"
+            )
+        );
+
+        final int exitCode = new CommandLine(
+            new ModrinthCommand()
+        )
+            .setExpandAtFiles(false)
+            .execute(
+                "--api-base-url", wm.getRuntimeInfo().getHttpBaseUrl(),
+                "--output-directory", tempDir.toString(),
+                "--game-version", "1.19.2",
+                "--loader", "fabric",
+                "--projects", "@" + listingFile
```

Comment:
The need for `@@` is what worried me. That would have been what users of the image would need to also do. Instead, using `--arg=value` syntax seems to bypass the at-file processing of picocli. So, changing to this seems best:

```suggestion
            .execute(
                "--api-base-url", wm.getRuntimeInfo().getHttpBaseUrl(),
                "--output-directory", tempDir.toString(),
                "--game-version", "1.19.2",
                "--loader", "fabric",
                "--projects=@" + listingFile
```